homeassistant/components/thermostat/__init__.py

```diff
@@ -246,6 +251,11 @@ def is_fan_on(self):
         """Return true if the fan is on."""
         return None
 
+    @property
+    def thermostat_mode(self):
```

Comment:
That's not what I intended it to do…

It's about what it is doing. I was actually considering if that should be the main state of a thermostat instead of the target temp.